cmd/opm/alpha/generate/generate.go

```diff
@@ -0,0 +1,117 @@
+package generate
```

Comment:
I think my preference would be to call this "build" or "save" instead of "generate" since that aligns more with the docker/podman CLIs that folks are familiar with. WDYT?